unique.c

```diff
@@ -0,0 +1,37 @@
+/*Implement an algorithm to determine if a string has all unique characters. Name this program unique.c.*/
+#include <stdio.h>
+#include <string.h>
+int main()
+{
+	char s[100];
+	int x;
+	printf("Enter text:\n");
+	fgets(s, sizeof(s), stdin);
+	sscanf(s, "%d", &x);
+	int length=strlen(s);
+	int i=0;
+	int add=0;
+	int counter=0;
+/*	printf("length is %d\n", length);*/
+	for(i=0; i<length; i++)
+	{				/*these two for loops rotate the numbers being compared*/
+	if (s[i]==' ') 			/*makes program ignore spaces*/
```

Comment:
Nice job! 💯 I see you took into consideration that the string might be a sentence - awesome job! 💯